README.md

```diff
@@ -1,32 +1,344 @@
-# Implementing Relations In SQL
+# Implementing Table Relations in SQL
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Implement a one-to-many relationship using an FOREIGN KEY constraint.
+- Implement a many-to-many relationship using FOREIGN KEY constraints along with a composite PRIMARY KEY constraint.
+- Implement a many-to-many relationship using FOREIGN KEY constraints, a single column PRIMARY KEY constraint, and
+  a composite UNIQUE constraint.
+- Implement a one-to-one relationship using a FOREIGN KEY and a UNIQUE constraint.
+
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In this lesson, we use SQL to implement the various relationships between tables.
+
+## Implementing One-To-Many Relationships in SQL
+
+![employee department ERD](https://curriculum-content.s3.amazonaws.com/6036/introduction-to-table-relations/dept_emp_erd.png
+)
+
+We will see how to use SQL to implement the one-to-many relationship between `employee` and `department`.
+Recall our sample data:
+
+<table>
+<tr>
+<th>department</th>
+<th>employee</th>
+</tr>
+
+<tr>
+
+<td>
+
+| id  | name      | location   |
+|-----|-----------|------------|
+| 1   | Payroll   | Building A |
+| 2   | Marketing | Building B |
+
+</td>
+
+<td>
+
+| id  | first_name | last_name  | salary | department_id |
+|-----|------------|------------|--------|---------------|
+| 1   | Kiran      | Willow     | 76000  | 2             |
+| 2   | Dani       | Elm        | 99000  | 2             |
+| 3   | Hao        | Pine       | 45000  | 2             |
+| 4   | Tal        | Oak        | 88000  | 1             |
+| 5   | Yuri       | Birch      | 150000 | 1             |
+
+</td>
+
+
+</tr>
+</table>
+
+- The  `department_id` column stores a single integer, thus each employee works in one department.
+- The `department_id` column is **not** unique. Multiple rows in the `employee` table may have the same value in the `department_id` column,
+  thus a department may have multiple employees.
+
+We implement the relationship using a `FOREIGN KEY` constraint on the `department_id` column in the `employee` table:
+
+```sql
+DROP TABLE IF EXISTS department CASCADE;
+
+CREATE TABLE department (
+  id INTEGER PRIMARY KEY,
+  name TEXT,
+  location TEXT
+);
+
+DROP TABLE IF EXISTS employee;
+
+CREATE TABLE employee (
+  id INTEGER PRIMARY KEY,
+  first_name TEXT,
+  last_name TEXT,
+  salary INTEGER,
+  department_id INTEGER NOT NULL,
+  CONSTRAINT department_fk FOREIGN KEY (department_id) REFERENCES department (id)
+);
+```
+
+The foreign key forces the database to check that each value in
+the `department_id` column corresponds to a value in the `id` column
+of the `department` table.  Thus, the following statement would
+result in a foreign key constrain violation as the `department` table
```

Comment:
Typo: "constrain" should be "constraint"

Reply:
fixed